Update DCP_change_log.md (#3974)
SkylineCommunications/dataminer-docs: release-notes/Cloud/DCP_change_log.md

### 26 November 2024 - New feature - Admin app - Automation usage [ID 41554]

From now on, the usage page in the [Admin app](https://admin.dataminer.services) will also provide usage data about Automation script runs when available.

### 26 November 2024 - New feature - Usage API - Usage API with API key [ID 41554]

From now on, you can create an API key on organization level with the "Retrieve usage data" permission. This API key can be used with the new Key Usage API, in combination with the new Public Usage API, to retrieve usage data about your DataMiner Systems in an automated way.

The swagger documentation pages about the available Usage API calls are available in the following locations:

- [Public Usage API swagger documentation](https://api.dataminer.services/swagger/usageapi/index.html?urls.primaryName=Public+Usage+Api+v1.0)
  - Get the features for which usage data might be available.
  - Get the metrics of a feature.
- [Key Usage API swagger documentation](https://api.dataminer.services/swagger/usageapi/index.html?urls.primaryName=Key+Usage+Api+v1.0)

>   - Get the data in a given time range for a given feature, metric and granularity with the option to filter and split data on the dimensions or DataMiner Systems.

Reviewer: Could you clarify this sentence? It isn't clear to me what you mean with "metric and granularity" and "data on the dimensions".

Author: these are inputs for the api call 
![image](https://github.com/user-attachments/assets/0d38bc2c-4865-4feb-883d-619192c5ec2c)

Feature examples: Automation, Storage as a Service,...
Metric examples: Script Runs, Operations,....
Granularity options: Daily, Weekly or Monthly
Dimension splitter examples: split usage data on 'Script Name', split usage data on script 'Succeeded', ...

### 26 November 2024 - Enhancement - Admin app - Usage and audit export email layout [ID 41554]

> From now on, the emails with the download link for usage exports or audit exports will be in the familiar email layout.

Reviewer: What do you mean with "the familiar email layout"? Could you rephrase this so it's also clear for people who might not know this?

Author: This email existed but as plain text without template/layout like our other emails do (like live sharing emails, expiring daas emails etc)

Reviewer: ```suggestion
From now on, the emails with the download link for usage exports or audit exports will use the same template as other emails sent from dataminer.services.
```

> [!NOTE]
> Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).

### 26 November 2024 - New Feature - Admin app - Automation Usage [ID ]

> From now on, the usage page in the [Admin app](https://admin.dataminer.services) will provide usage data about the Automation script runs when available.

Author: ```suggestion
From now on, the usage page in the [Admin app](https://admin.dataminer.services) will provide usage data about Automation script runs when available.
```

### 26 November 2024 - New Feature - Usage API - Usage API with API key [ID ]

From now on, you can create an API key on organization level with the "Retrieve usage data' permission. This API key can be used with the new Key Usage API, in combination with the new Public Usage API, to retrieve usage data about your DataMiner Systems in an automated way.

Find the swagger documentation pages about the available Usage API calls via the following references:
- [Public Usage API swagger documentation](https://api.dataminer.services/swagger/usageapi/index.html?urls.primaryName=Public+Usage+Api+v1.0)
  - Get the features were usage data might be available for
  - Get the metrics of a feature
- [Key Usage API swagger documentation](https://api.dataminer.services/swagger/usageapi/index.html?urls.primaryName=Key+Usage+Api+v1.0)
  - Get the data in a given time range for a given feature, metric and granularity with the option to filter & split data on the dimensions or DataMiner Systems

> ### 26 November 2024 - Enhancement - Admin app - Usage export to CSV email layout [ID ]

Author: ```suggestion
### 26 November 2024 - Enhancement - Admin app - Usage export email layout [ID ]
```

From now on, the emails with the download link for usage exports will be in the familiar email layout.

> ### 26 November 2024 - Enhancement - Admin app - Audit export to CSV email layout [ID ]

Author: ```suggestion
### 26 November 2024 - Enhancement - Admin app - Audit export email layout [ID ]
```
